frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view_helpers.ts

```diff
@@ -287,3 +292,68 @@ export function findParentGroupNode(nodes: TreeNode[], parentGroupId: number): T
   });
   return foundParentNode;
 }
+
+export const registerSegmentsForBoundingBox = async (value: Vector6, bbName: string) => {
```

Comment:
I think, this code should be moved into the `TracingApi` in `api_latest.ts`.

Reply:
the Vector6 interface is not very user-friendly in my opinion (because it's not clear whether the second vec3 is the extent or the max of the bounding box). why not two separate variables? either min and max or min and extent?